provider/resource_harbor_project.go

```diff
@@ -36,6 +36,11 @@ func resourceProject() *schema.Resource {
 				Optional:    true,
 				Default:     false,
 			},
+			"projectid": {
```

Comment:
why is this attribute needed?  the id for this resource should already contain the information you need in order to delete artifacts from this project.

Reply:
Currently the Id field is being set as `/projects/${projectid}`. I thought that adding another attribute that was just the number would be cleaner than parsing the Id string

Reply:
I'd prefer if we didn't add an extra attribute only for this purpose.  The point of a resource ID in terraform is to contain all of the information needed to make API calls like these, so it's okay to parse this string to get what you're looking for.

Reply:
Fixed